database/postgres/postgres.go

```diff
@@ -266,13 +266,25 @@ func runesLastIndex(input []rune, target rune) int {
 	return -1
 }
 
+func quoteIdentifierWithSchema(name string) string {
+	if strings.Index(name, ".") != -1 {
```

Comment:
Let's make this optional by adding a `x-migrate-table-has-schema` and default to `false`. Otherwise, existing customized migration table names with a `.` character in it would break.

Reply:
@dhui done